syntax = "proto3";

package access;

service AccessControl {
  rpc CheckUID (UIDRequest) returns (AccessResponse);
}

message UIDRequest {
  string uid = 1;
}

message AccessResponse {
  bool is_authorized = 1;
}
